### ISSUE TITLE
Android Returning from any Url Sets Nav bar icons to white

### ISSUE THREAD

**Issue by workerbee22** [Western-Health-Covid19-Collaboration/wh_covid19_app#307]
On Android, for any and all urls, when returning to the app from a url, results in the returned screen nav bar icons being white. On some Android versions (ie. v8.x) this results in white on white nav bar, other Android versions (ie. v10.x) results in white on grey nav bar as system adapts. Expected result is the app nav and status bar themes ie. currently dark icons on white background.

Android 8.x

![image](https://user-images.githubusercontent.com/5154664/80673743-5ec6e300-8af3-11ea-89ba-132b3f9bf2fa.png)

Android 10.x

![image](https://user-images.githubusercontent.com/5154664/80673763-6f775900-8af3-11ea-99db-b7f04909164c.png)

**Comment by lukesleeman** [comment]
Yeah, I have noticed this as well!  👏 great job on getting it into the issue tracker and figuring out how to reproduce it.

Do you think we need to address this for V1?

**Comment by workerbee22** [comment]
Optional for v1 ... it just bugs me and I think easy fix.

**Comment by lukesleeman** [comment]
This still seems to be present
